src/main/resources/label-definitions.properties

```diff
@@ -1068,6 +1068,7 @@ wwpass-plugin=user
 xcp-ci=external
 xfpanel=report ui
 xframe-filter-plugin=page-decorator misc
+xlrelease-plugin=platform post-build external
```

Comment:
Is this intentionally adding an unrecognized category? It will show as `Misc (platform)` in Jenkins.

Reply:
@daniel-beck Thanks for noticing it, just removed it, for now will add some meaningful labels later